apps/dashboard/src/app/components/search-filters/search-filters.module.ts

```diff
@@ -9,6 +9,7 @@ import { STFExpanderModule } from '@flaps/pastanaga';
 import { SearchFiltersComponent } from './search-filters/search-filters.component';
 import { ButtonClearComponent } from './button-clear/button-clear.component';
 import { FilterOntologyComponent } from './filter-ontology/filter-ontology.component';
+import { PaIconModule } from '@guillotinaweb/pastanaga-angular';
 
 @NgModule({
```

Comment:
I think SearchFilterModule is not used anymore, it corresponds to an old design. Maybe it can be deleted at some point.

Reply:
Do you think I can delete it in this PR? I don't like keeping dead code in a repo

Reply:
yes

Reply:
cool, done. Thanks